backend/pyproject.toml

```
@@ -33,6 +33,7 @@ dependencies = [
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
     "unstract-sdk~=0.56.0rc4",
+    "gcsfs~=2024.10.0",
```

Unpinning here does not work is it?

---

@gaya3-zipstack please unpin at minor version level like below at every place

```suggestion
    "gcsfs~=2024.10",
```

---

@gaya3-zipstack also looking at this [commit](https://github.com/Zipstack/unstract/pull/1106/commits/9d5c60ba23b94dd54bedff340ab71741d886ec11) 

I think we should keep the min version to 6. For backend I think that is the only compatible version. So keep it 
 ```suggestion
    "gcsfs~=2024.6",
```

and try